state_machine/applications/system-check/tests/sun_sensor_test.py

```diff
@@ -0,0 +1,174 @@
+"""
+Python system check script for PyCubed satellite board
+PyCubed Mini mainboard-v02 for Pocketqube Mission
+Sun Sensor Test
+* Author(s): Yashika Batra
+"""
+
+import time
+from lib import pycubed as cubesat
+
+
+def lux(sun_sensor_index):
+    """ Return the lux reading for a given sun sensor """
+    if sun_sensor_index == "-Y":
+        return cubesat._cubesat.sun_yn.lux
+    elif sun_sensor_index == "-Z":
+        return cubesat._cubesat.sun_zn.lux
+    elif sun_sensor_index == "-X":
+        return cubesat._cubesat.sun_xn.lux
+    elif sun_sensor_index == "+Y":
+        return cubesat._cubesat.sun_yp.lux
+    elif sun_sensor_index == "+Z":
+        return cubesat._cubesat.sun_zp.lux
+    elif sun_sensor_index == "+X":
+        return cubesat._cubesat.sun_xp.lux
+    else:
+        print(sun_sensor_index, "is not a defined sun sensor.")
+        return
+
+
+def user_test(sensor_index, light_dark):
+    """
+    All user interaction happens in this function
+    Set wait times, prompt the user and collect lux data
+    """
+
+    wait_time = 3
+
+    print("Testing Sun Sensor {sensor_index} in the {light_dark}.")
+
+    if light_dark == "Light":
+        print("Please turn on the lights. Waiting {wait_time} seconds.")
+    elif light_dark == "Dark":
+        print("Please remove all nearby light sources or cover the sensor. " +
+              "Waiting {wait_time} seconds.")
+
+    time.sleep(wait_time)
+    print("Collecting sensor data...")
+    lux_val = lux(sensor_index)
+    print("Data collection complete.")
+    return lux_val
+
+
+def test_sensorx(result_dict, sensor_index, light_dark):
```

Comment:
don't love the name of this function (not very descriptive). How about `test_individual_sun_sensor`?